app/javascript/packs/Message/message_form.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Form, Button } from 'react-bootstrap';
+import axios from 'axios';
+
+
+export default class CreateForm extends React.Component {
+    constructor(props, context) {
+      super(props, context);
+  
+      this.state = {
+        maxLength: 200,
+        show: false,
+        content: "",
+        postID: this.props.notifID,
+        errorDescription: ""
+      };
+    }
+
+    handleSubmit = (e) => {
+      e.preventDefault();
+      const data = new FormData();
+
+      data.append('message[content]', this.state.content)
+      data.append('message[post_event_id]', this.state.postID)
+
+      axios.post("/messages.json", data, 
+      {headers: {
+          "X-CSRF-Token": document.querySelector('meta[name="csrf-token"]').content
+      }}).then(() =>
+        this.props.fetchMessages()
+      ).then(() =>
+        document.getElementById('textarea').value = ''
```

Comment:
you shouldn't do any operations directly on DOM. Use react state to clear textarea.